### ISSUE TITLE
bert score breaks with current latest version of matplotlib

### ISSUE THREAD

**Issue by mattshardlow** [feralvam/easse#95]
If I try to do:

> from bert_score import Bert_Scorer

I get

> AttributeError: module 'matplotlib.cbook' has no attribute '_make_class_factory'

I can fix this by installing matpotlib 3.4.3, in which case the error goes.

I can put 3.4.3 in the matplotlib reqs for EASSE, but really this should be solved in the BertScore reqs. I'm not sure if putting it in bertscore's reqs will propagate through to EASSE on instalation though, or if easse's req for the latest matplotlib will override this. Any strong opinions on where to put this?

**Comment by louismartin** [comment]
For now we can hotfix it by putting 3.4.3 in EASSE reqs (I will accept the PR if you want to go ahead).
The ideal solution would indeed be to put it in BertScore reqs and I think it should propagate to EASSE. Even better, maybe you should raise the issue in BertScore so that the issue with `_make_class_factory` is fixed directly in BertScore.
Does that make sense?

**Comment by mattshardlow** [comment]
Yeah, I think it's better to try and get it accepted over there. I'll raise it and  see if they accept in a reasonable timeframe. If not, I'll issue the PR here and that hopefully won't break everything else.
